doc/source/wsrep-system-index.rst

```diff
@@ -292,18 +292,40 @@ Defines ``DBUG`` options to pass to the wsrep provider.
    :conf: Yes
    :scope: Global
    :dyn: Yes
-   :default: ``OFF``
+   :default: ``NONE``
+
+Enables debug level logging for the database server and `wsrep-lib
+<https://github.com/codership/wsrep-lib>`_ - an integration library for WSREP
+API with additional convenience for transaction processing.  By default,
+:variable:`--wsrep-debug` is disabled.
 
-Enables additional debugging output for the database server error log.
-By default, it is disabled.
-This variable can be used when trying to diagnose problems
-or when submitting a bug.
+This variable can be used when trying to diagnose problems or when submitting a
+bug.
 
 You can set ``wsrep_debug`` in the following :file:`my.cnf` groups:
 
 * Under ``[mysqld]`` it enables debug logging for ``mysqld`` and the SST script
 * Under ``[sst]`` it enables debug logging for the SST script only
 
+This variable may be set to one of the following values:
+
+NONE
+   No debug-level messages.
+
+SERVER
+
+   debug-level messages from wsrep-lib *server_state* and *others* parts. Galera
```

Comment:
'others' refer to the diagram I attached to the Jira ticket. In my opinion it can be not clear for the user what is it.
Maybe it is more clear to say here:
SERVER - wsrep-lib general debug-level messages and detailed debug-level messages from 'server_state' part are printed out. 
Galera debug-level logs are printed out.

TRANSACTION -  Same as SERVER + wsrep-lib *transaction* part

... and so on